src/dawn/Compiler/DawnCompiler.cpp

```diff
@@ -165,6 +166,7 @@ std::unique_ptr<OptimizerContext> DawnCompiler::runOptimizer(std::shared_ptr<SIR
   optimizer->checkAndPushBack<PassSetBoundaryCondition>();
   optimizer->checkAndPushBack<PassDataLocalityMetric>();
   optimizer->checkAndPushBack<PassSetSyncStage>();
+  optimizer->checkAndPushBack<PassSetBlockSize>();
```

Comment:
I would prefe it to be before DataLocalityMetric

Reply:
ok